spec/kitchen/driver/gce_spec.rb

```diff
@@ -92,7 +92,8 @@
         zone_name: nil,
         google_key_location: nil,
         google_json_key_location: nil,
-        preemptible: false
+        preemptible: false,
+        auto_restart: false
```

Comment:
You can set `auto_restart` to `true` here, right? That's a more reasonable default as for most users, if they want non-preemptible VMs, they want the VMs to auto-restart on their own.